src/vim-qflist-json-error-reporter.cpp

```diff
@@ -120,17 +120,25 @@ vim_qflist_json_error_formatter::vim_qflist_json_error_formatter(
 
 void vim_qflist_json_error_formatter::write_before_message(
     severity sev, const source_code_span &origin) {
-  if (sev == severity::note) {
+  std::string_view severity_value{};
```

Comment:
Nit: Remove `{}`. That's not the style we use in this code base.
```suggestion
  std::string_view severity_value;
```